weather_man.py

```diff
@@ -0,0 +1,142 @@
+import os
+import glob
+import csv
+import argparse
+from calculations import do_calculations
```

Comment:
Does `do_calculations` make sense? What calculations they are? What they are going to perform?